model_external_socket/java/test/org/openda/externalsocket/ExternalModelStochModelFactoryTest.java

```diff
@@ -0,0 +1,33 @@
+package org.openda.externalsocket;
+
+import junit.framework.TestCase;
+import org.openda.interfaces.IStochModelFactory;
+import org.openda.interfaces.IVector;
+import org.openda.utils.OpenDaTestSupport;
+
+import java.io.File;
+
+public class ExternalModelStochModelFactoryTest extends TestCase {
+
+	private File testRunDataDir;
+	private OpenDaTestSupport testData;
```

Comment:
Codacy found an issue: [Perhaps 'testData' could be replaced by a local variable.](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=6157987)